views/020_teams_triggers.sql

```diff
@@ -1,14 +1,26 @@
+-- Notify on any updates on the teams table
 CREATE OR REPLACE FUNCTION handle_team_updates()
 RETURNS TRIGGER AS $$
 BEGIN
+  IF TG_OP = 'DELETE' THEN
+    PERFORM pg_notify('table_activity', TG_TABLE_NAME || ' ' || OLD.id);
```

Comment:
Created a new NOTIFY channel `table_activity` which should be used to publish any table updated notify instead of having multiple notify channels like `teams_updated` or `notifications_updated`.

This way we can use the notify router and have one listener on the NOTIFY channel and then distribute to multiple consumers based on the table name.

Reply:
The payload should be in the format `<route> <an optional payload>`. The route is used by the pg notify router and the remaining payload is sent to the consumers.